src/components/ScoreBoard/Player/Player.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Counter from "../Counter/Counter";
+
+const Player = (props) => <div className="player">
+  <span className="player-name">
+    <button
+      className="remove-player"
+      onClick={() => props.removePlayer(props.id)}
+    >
+      ✖
+    </button>
+    { props.name }
+  </span>
+
+  <Counter />
+</div>;
```

Comment:
For multiple lines of JSX codes, we should keep the body parenthesis. It's much easier to read that way.
```
const Player = (props) => (
  <div className="player">
    <span className="player-name">
      <button
        className="remove-player"
        onClick={() => props.removePlayer(props.id)}
      >
      ✖
      </button>
      {props.name}
    </span>

    <Counter />
  </div>
);
```